include/libopencm3/efm32/hg/gpio.h

```diff
@@ -2,7 +2,6 @@
  * This file is part of the libopencm3 project.
  *
  * Copyright (C) 2015 Kuldeep Singh Dhaka <kuldeepdhaka9@gmail.com>
- * Copyright (C) 2018 Seb Holzapfel <schnommus@gmail.com>
```

Comment:
Try and squish this sort of commit back out of existance, rather than introducing code early in a series and promptly deleting it.  Just don't add it in the first place.

Reply:
Sure thing, I should have checked out the other PRs and noticed you were doing per-commit reviews rather than looking at all the changes at once. Cheers!